mkdocs.yml

```diff
@@ -145,5 +168,15 @@ nav:
   #   - integrations/index.md
   - API Reference:
     - Python SDK: reference/argilla_sdk/
+  - Community:
+    - community/index.md
+    - Contributor Guide: community/contributor.md
+    - Developer Guide: community/developer.md
+    - Frequently Asked Questions (FAQs): community/faq.md
+    - About us: community/about.md
+    - Changelog ↗: https://github.com/argilla-io/argilla/blob/develop/argilla/CHANGELOG.md
+    - Slack ↗: https://join.slack.com/t/rubrixworkspace/shared_invite/zt-whigkyjn-a3IUJLD7gDbTZ0rKlvcJ5g
+    - Community Meetup ↗: https://lu.ma/argilla-event-calendar
+    - Roadmap ↗: https://github.com/orgs/argilla-io/projects/10/views/1
```

Comment:
Not sure if I agree. I decided to do it this way to keep to main page grid aligne with the sidebar pagination to avoid confusion. What do other people think?

Reply:
@sdiazlor Mentions it here: https://github.com/argilla-io/argilla-python/pull/224#pullrequestreview-2084898601

Just to add more context. IMO, there is value in the grids plus the menu because the menus gives core items and the grid gives more information. I don't think the two need to align completely because we end up in a everything-everywhere situation.